modules/op-creating-webhooks.adoc

```diff
@@ -0,0 +1,41 @@
+// This module is included in the following assembly:
+//
+//  *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="creating-webhooks_{context}"]
+= Creating Webhooks
+
+_Webhooks_ are HTTP POST messages that are received by the EventListeners whenever a configured event occurs in your repository. The event payload is then mapped to TriggerBindings, and processed by TriggerTemplates. The TriggerTemplates eventually start one or more PipelineRuns, leading to the creation and deployment of Kubernetes resources.
+
+In this section you will configure a Webhook URL on your forked Git repositories `vote-ui` and `vote-api`. This URL points to the publicly accessible EventListener service route.
+
+[NOTE]
+====
+Adding Webhooks requires administrative privileges to the repository. If you do not have administrative access to your repository, contact your System Administrator for adding Webhooks.
```

Comment:
s/System Administrator/system administrator/
This is a generic title, not a specific proper noun.